internal/service/streamprocessor/resource.go

```diff
@@ -17,6 +17,11 @@ const StreamProcessorName = "stream_processor"
 
 var _ resource.ResourceWithConfigure = &streamProcessorRS{}
 var _ resource.ResourceWithImportState = &streamProcessorRS{}
+var (
+	errorDuringCreateStartActions    = "You need to fix the processor and import the resource or delete it manually and re-run terraform apply."
```

Comment:
knit: I'd remove "During" to make it more similar to error const in other resources

Reply:
Thanks. Changed in 58707c352efb05185f548944ffab927eee535bf0